{
	"documenttitle": {
		"text": "Red Hat Security Advisory: Django security update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2013:0670"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2013-03-21T18:12:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2013-03-21T18:11:00Z",
		"CurrentReleaseDate": "2013-03-21T18:12:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.0.0",
			"Date": "2021-08-23T04:38:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nUpdated Django packages that fix multiple security issues are now available\nfor Red Hat OpenStack Folsom.\n\nThe Red Hat Security Response Team has rated this update as having moderate\nsecurity impact. Common Vulnerability Scoring System (CVSS) base scores,\nwhich give detailed severity ratings, are available for each vulnerability\nfrom the CVE links in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nThe Django web framework is used by Horizon, the OpenStack Dashboard, which\nis a web interface for managing OpenStack services.\n\nA denial of service flaw was found in the Extensible Markup Language (XML)\nparser used by Django. A remote attacker could use this flaw to send a\nspecially-crafted request to an Horizon API, causing Horizon to consume an\nexcessive amount of CPU and memory. (CVE-2013-1664)\n\nA flaw was found in the XML parser used by Django. If a remote attacker\nsent a specially-crafted request to an Horizon API, it could cause Horizon\nto connect to external entities, causing a large amount of system load, or\nallow an attacker to read files on the Horizon server that are accessible\nto the user running Horizon. (CVE-2013-1665)\n\nIt was found that the history view in the bundled administrative web\ninterface in Django did not perform permission checks. An authenticated\nuser who can access the web interface could use this flaw to view the\nchanges to objects and data they would otherwise not have access to.\n(CVE-2013-0305)\n\nA flaw was found in the way Django handled formsets. A remote attacker\ncould use this flaw to submit a large number of forms, causing excessive\nmemory consumption. Note: The current version of OpenStack in Red Hat\nOpenStack Folsom does not expose this vulnerability. (CVE-2013-0306)\n\nThis update also includes two hardening fixes to help protect against XML\nentity and HTTP Host header poisoning attacks. (BZ#913039, BZ#913037)\n\nAll users of Horizon are advised to upgrade to these updated packages,\nwhich resolve these issues. After installing the updated packages, the\nhttpd daemon must be restarted (\"service httpd restart\") for the update to\ntake effect.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2021 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Moderate",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2013:0670",
				"Description": "https://access.redhat.com/errata/RHSA-2013:0670"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#moderate",
				"Description": "https://access.redhat.com/security/updates/classification/#moderate"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Django: Data leakage via admin history log",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2013-02-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2013-0305",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously-released errata\nrelevant to your system have been applied.\n\nThis update is available via the Red Hat Network. Details on how to\nuse the Red Hat Network to apply this update are available at\nhttps://access.redhat.com/knowledge/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2013:0670"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2013-0305",
						"Description": "CVE-2013-0305"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=913041",
						"Description": "bz#913041: CVE-2013-0305 Django: Data leakage via admin history log"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2013-02-19T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"6Server-Folsom:Django14-0:1.4.4-1.el6ost"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "4.0",
					"Vector": "AV:N/AC:L/Au:S/C:P/I:N/A:N"
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Django: Formset denial-of-service",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2013-02-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2013-0306",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously-released errata\nrelevant to your system have been applied.\n\nThis update is available via the Red Hat Network. Details on how to\nuse the Red Hat Network to apply this update are available at\nhttps://access.redhat.com/knowledge/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2013:0670"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2013-0306",
						"Description": "CVE-2013-0306"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=913042",
						"Description": "bz#913042: CVE-2013-0306 Django: Formset denial-of-service"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2013-02-19T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"6Server-Folsom:Django14-0:1.4.4-1.el6ost"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "5.0",
					"Vector": "AV:N/AC:L/Au:N/C:N/I:N/A:P"
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Python xml bindings: Internal entity expansion in Python XML libraries inflicts DoS vulnerabilities",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2013-02-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2013-1664",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously-released errata\nrelevant to your system have been applied.\n\nThis update is available via the Red Hat Network. Details on how to\nuse the Red Hat Network to apply this update are available at\nhttps://access.redhat.com/knowledge/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2013:0670"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2013-1664",
						"Description": "CVE-2013-1664"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=913808",
						"Description": "bz#913808: CVE-2013-1664 Python xml bindings: Internal entity expansion in Python XML libraries inflicts DoS vulnerabilities"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2013-02-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"6Server-Folsom:Django14-0:1.4.4-1.el6ost"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "4.3",
					"Vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Python xml bindings: External entity expansion in Python XML libraries inflicts potential security flaws and DoS vulnerabilities",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2013-02-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2013-1665",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying this update, make sure all previously-released errata\nrelevant to your system have been applied.\n\nThis update is available via the Red Hat Network. Details on how to\nuse the Red Hat Network to apply this update are available at\nhttps://access.redhat.com/knowledge/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2013:0670"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2013-1665",
						"Description": "CVE-2013-1665"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=912982",
						"Description": "bz#912982: CVE-2013-1665 Python xml bindings: External entity expansion in Python XML libraries inflicts potential security flaws and DoS vulnerabilities"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2013-02-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"6Server-Folsom:Django14-0:1.4.4-1.el6ost"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "5.8",
					"Vector": "AV:N/AC:M/Au:N/C:P/I:N/A:P"
				}
			}
		}
	],
	"producttree": {
		"xmlns": "http://www.icasi.org/CVRF/schema/prod/1.1",
		"branch": [
			{
				"type": "Product Family",
				"name": "Red Hat OpenStack Platform",
				"branch": [
					{
						"type": "Product Name",
						"name": "OpenStack Folsom",
						"fullproductname": {
							"text": "OpenStack Folsom",
							"productid": "6Server-Folsom",
							"cpe": "cpe:/a:redhat:openstack:2::el6"
						}
					}
				],
				"fullproductname": {}
			},
			{
				"type": "Product Version",
				"name": "Django14-0:1.4.4-1.el6ost",
				"fullproductname": {
					"text": "Django14-1.4.4-1.el6ost.src.rpm",
					"productid": "Django14-0:1.4.4-1.el6ost",
					"cpe": "cpe:/a:redhat:openstack:2::el6"
				}
			}
		],
		"relationship": [
			{
				"productreference": "Django14-0:1.4.4-1.el6ost",
				"relationtype": "Default Component Of",
				"relatestoproductreference": "6Server-Folsom",
				"fullproductname": {
					"text": "Django14-0:1.4.4-1.el6ost as a component of OpenStack Folsom",
					"productid": "6Server-Folsom:Django14-0:1.4.4-1.el6ost",
					"cpe": "cpe:/a:redhat:openstack:2::el6"
				}
			}
		]
	}
}
